src/commands/start.js

```diff
@@ -84,6 +110,8 @@ StartCommand.args = [{
 StartCommand.flags = {
   'full-node': flagTypes.boolean({ char: 'f', description: 'start as full node', default: false }),
   'operator-private-key': flagTypes.string({ char: 'p', description: 'operator private key', default: null }),
+  'drive-src-path': flagTypes.string({ char: 'r', description: 'drive source code path', default: null }),
```

Comment:
Could we remove `char` and require the whole name otherwise they kinda confusing?